components/properties/src/exemplar_chars.rs

```diff
@@ -172,14 +159,15 @@ make_exemplar_chars_unicode_set_property!(
     /// assert!(exemplars_punctuation.contains_char('!'));
     /// assert!(exemplars_punctuation.contains_char('?'));
     /// ```
-
-    pub fn load_exemplars_punctuation();
+    pub fn load_exemplars_punctuation_stable();
```

Comment:
Question: did you add the `_stable` suffix because the other function doesn't have a suffix?

(That was an intentional choice in ICU4X 1.0 because we felt that the data struct would be sufficiently stable, since it has an internal enum, so we wouldn't need Any/Buffer constructors)

Reply:
Stripping the `load_` and `_stable` for now